staging/src/k8s.io/apiextensions-apiserver/pkg/controller/status/naming_controller.go

```diff
@@ -62,6 +62,7 @@ type NamingConditionController struct {
 }
 
 func NewNamingConditionController(
+	ctx context.Context,
```

Comment:
https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/126387/pull-kubernetes-apidiff/1818538357228048384:
```
- ./pkg/controller/status.NewNamingConditionController: changed from func(k8s.io/apiextensions-apiserver/pkg/client/informers/externalversions/apiextensions/v1.CustomResourceDefinitionInformer, k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset/typed/apiextensions/v1.CustomResourceDefinitionsGetter) *NamingConditionController to func(context.Context, k8s.io/apiextensions-apiserver/pkg/client/informers/externalversions/apiextensions/v1.CustomResourceDefinitionInformer, k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset/typed/apiextensions/v1.CustomResourceDefinitionsGetter) *NamingConditionController
```

This seems to be [unused outside of Kubernetes](https://grep.app/search?q=NewNamingConditionController), so breaking the API seems acceptable.

Reply:
I think all controllers can be considered internal and breaking consumers is ok.